x/margin/keeper/msg_server.go

```diff
@@ -344,6 +344,23 @@ func (k msgServer) UpdatePools(goCtx context.Context, msg *types.MsgUpdatePools)
 	return &types.MsgUpdatePoolsResponse{}, nil
 }
 
+func (k msgServer) UpdateRowanCollateral(goCtx context.Context, msg *types.MsgUpdateRowanCollateral) (*types.MsgUpdateRowanCollateralResponse, error) {
```

Comment:
We can probably change the update params logic , instead of adding new transactions for each param ?